### DISCUSSION TITLE
Is `@repo` prefix for packages a standard/recommended?

### DISCUSSION THREAD

**Discussion by bitttttten** [vercel/turborepo#9649] [Help] [answered]
### Summary

I see examples in the docs with `{ "name": "@repo/ui" }` i.e. [in internal packages docs](https://turbo.build/repo/docs/core-concepts/internal-packages).

Is this a recommended standard? We are currently adopting this practise in our repo, but had some questions like why is it not `@{reponame}` as in, let's say our repo is called "frontend" why not `@frontend`?

**Comment by anthonyshew** [comment, marked as answer]
This is only a convention we use in our examples and documentation, since it's an unusable, unclaimable name on the npm registry. It carries no special behavior beyond that.
